internal/contour/virtualhost.go

```diff
@@ -28,6 +29,28 @@ type VirtualHostCache struct {
 	Cond
 }
 
+func getRequestTimeout(annotations map[string]string) *time.Duration {
```

Comment:
s/*time.Duration/time.Duration/

Reply:
discussed in chat, 0 duration sadly has a different meaning than "null pointer" / unset duration in envoy config